src/tests/test_browser_apis.py

```diff
@@ -224,6 +224,63 @@ def foo(obj):
     remove_event_listener(x, "click", foo)
 
 
+@run_in_pyodide
+async def test_additional_event_listener_arguments(selenium):
+    from pyodide.code import run_js
+
+    x = run_js(
+        """
+class MockObject {
+    constructor() {
+        this.listeners = {};
+    }
+    addEventListener(event, handler, arg1, arg2) {
```

Comment:
Why does the mock have a nonstandard signature?

Reply:
Adjusted to just use `kwargs`. Thanks!